src/transformers/models/bert_japanese/tokenization_bert_japanese.py

```diff
@@ -586,9 +589,15 @@ def __init__(
         else:
             raise ValueError("Invalid sudachi_split_mode is specified.")
 
-        self.sudachi = dictionary.Dictionary(
+        self.projection = sudachi_projection
+
+        sudachi_dictionary = dictionary.Dictionary(
             config_path=sudachi_config_path, resource_dir=sudachi_resource_dir, dict=sudachi_dict_type
-        ).create(self.split_mode)
+        )
+        if self.projection:
+            self.sudachi = sudachi_dictionary.create(self.split_mode, projection=self.projection)
+        else:
+            self.sudachi = sudachi_dictionary.create(self.split_mode)
```

Comment:
```suggestion
         self.sudachi = sudachi_dictionary.create(self.split_mode, projection=self.projection)
```
this should work no? Unless call with none is not supported?

Reply:
This causes exceptions. Previous versions of SudachiPy do not have `projection` arg for `Dictionary.create()`.

Reply:
I'd like to add a comment to describe above comment.